staging/src/k8s.io/client-go/tools/record/events_cache.go

```diff
@@ -144,7 +144,9 @@ func (f *EventSourceObjectSpamFilter) Filter(event *v1.Event) bool {
 	// update the cache
 	f.cache.Add(eventKey, record)
 
-	return filter
+	//if event reason is healthy and can get a token, return skip
+	return filter && event.Reason != events.ContainerHealthy
```

Comment:
I don't think it's a good idea to let these through without blocking something else.

Now that P&F is in beta, we could think about not having this limit and e.g. using two different user names or something so that healthy events can be more important. But that will require careful testing.